pkg/aws/client.go

```diff
@@ -289,8 +290,38 @@ func (c *awsClient) ValidateCFUserCredentials() error {
 
 	return fmt.Errorf(
 		"Invalid CloudFormation stack credentials: %s is not valid \n"+
-			"you can recreate the CloudFormation stack with: \n"+
-			"moactl init --delete-stack && moactl init \n", name)
+			adviseRecreate(), name)
+}
+
+func (c *awsClient) EnsureOsdCcsAdminUserPermissions() error {
+	AdminUserNamePtr := AdminUserName
```

Comment:
`AdminUserName` is actually a const and can't be used as a pointer so that another object needs to be created :)) you can have a look at the rest of the file and will come across several definements like that.. Is there a better way of getting rid of it?

Reply:
I've changed it to `adminUserName` now, thanks!